tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1384,9 +1395,7 @@ def check_return(self, node):
         function_decorators = node.decoratornames()
         try:
             returns = next(node.infer_call_result())
-            # If returns None, or raises an error ignore
```

Comment:
Discussed and decided to allow for users to locally disable `raise` only functions